src/lib/openapi/spec/playground-response-schema.ts

```diff
@@ -0,0 +1,34 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { sdkContextSchema } from './sdk-context-schema';
+import { playgroundRequestSchema } from './playground-request-schema';
+import { playgroundFeatureSchema } from './playground-feature-schema';
+
+export const playgroundResponseSchema = {
+    $id: '#/components/schemas/playgroundResponseSchema',
+    description: 'The state of all toggles given the provided input.',
+    type: 'object',
+    additionalProperties: false,
+    required: ['toggles', 'input'],
+    properties: {
+        input: {
+            $ref: playgroundRequestSchema.$id,
+        },
+        toggles: {
```

Comment:
I think more verbose "featureToggles" is more fitting.

Reply:
I think @andreas-unleash said that `features` is the new standard. Changed it to that in f085dcc653c4ddd1de77f3b63147e0121dfef667; would be happy to change it to `featureToggles` if that's what we're going with too.